### PR TITLE
Delete

### PR DESCRIPTION
**Pull request by danicadombek**
Create DeleteButton-component and add it to MemoriesPage with logic.

### CODE REVIEW

**Review by Morten-Moeller — APPROVED** [review]
Looks good, keep on rolling! 👾

**Review by christofjens — COMMENTED** [review]
LGTM!
Some file/function names are a bit hard to follow, like `ToDeleteButton` in the `DeleteButton.js` or `handleDeleteMemory()` and `handleMemorySubmit()`, but otherwise it looks fine. Clean coding and well to read. If no one with more experience is reviewing, you're good to go anyway. 👍
